src/main/java/edu/greenblitz/robotName/commands/LED/Rumble.java

```diff
@@ -0,0 +1,34 @@
+package edu.greenblitz.robotName.commands.LED;
+
+import edu.greenblitz.robotName.OI;
+import edu.greenblitz.robotName.subsystems.LED.LEDConstants;
+import edu.greenblitz.robotName.utils.GBCommand;
+import edu.wpi.first.wpilibj.Timer;
+
+public class Rumble extends GBCommand {
+	private Timer timer;
+	
+	public Rumble() {
+		timer = new Timer();
+	}
+	
+	@Override
+	public void initialize() {
+		timer.restart();
+	}
+	
+	@Override
+	public void execute() {
+		OI.getInstance().getMainJoystick().rumble(LEDConstants.RUMBLE_LEFT_MOTOR, LEDConstants.RUMBLE_POWER);
```

Comment:
consider saving a variable `mainJoystick` instead of calling the `getInstance()` all of the time